SwanCustomEnvironments/swancustomenvironments/scripts/builders/accpy.sh

```diff
@@ -6,6 +6,20 @@ if [ -n "${INSTALL_NXCALS}" ]; then
     SPARKMONITOR="sparkmonitor==$(python -c 'import sparkmonitor; print(sparkmonitor.__version__)')"
```

Comment:
In the message of this commit, we need to explain that we are creating an environment to host SWAN packages for Spark (aka middle layer). The purpose of such SWAN environment is to have a user environment on top with only user packages, which in case of being pip freezed it will contain the version and dependency resolution for user packages only